packages/rancher-alerting-drivers/charts/Chart.yaml

```diff
@@ -0,0 +1,44 @@
+name: rancher-alerting-drivers
+home: ""
+sources: []
+version: 1.0.1
+description: The manager for third-party webhook receivers used in the Alertmanger
+keywords:
+- monitoring
+- alertmanger
+- webhook
+maintainers:
+- name: Jiaqi Luo
+  email: ""
+  url: https://github.com/jiaqiluo
+icon: ""
+apiversion: v2
+condition: ""
+tags: ""
+appversion: 1.16.0
+deprecated: false
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Alerting Drivers
+  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/release-name: rancher-alerting-drivers
```

Comment:
We could add requires-gvr here, but its not actually necessary for this to functionally run, so we should just leave off. That is unless it requires any monitoring CRDs ?

Reply:
This app does not have any dependencies on monitoring CRDs.